README.md

```diff
@@ -76,6 +76,25 @@ and execute the following command in the root of the repository:
 
 The build files are found in the `book` directory.
 
+### Pre-commit script
+
+We also test that line lengths are less than 100 columns. To test this locally,
+you can run `ci/check_line_lengths.sh`.
+
+You can also set this to run automatically.
+
+On Linux:
+
+```bash
+ln -s ../../ci/check_line_lengths.sh .git/hooks/pre-commit
+```
+
+On Windows:
+
+```powershell
+cd .git/hooks && New-Item -Path ../../.git_hooks/* -ItemType SymbolicLink -Value . && cd ../..
+```
+
```

Comment:
This note would also be helpful.

```suggestion

Note that you should run PowerShell as an administrator to create a symbolic link.

```

Reply:
> Note that you should run PowerShell as an administrator to create a symbolic link.

Hmm, maybe we could use hard links instead? Will that update the hook if `ci/check_line_lengths.sh` is updated?

Reply:
Both reflect changes from the original file (`ci/check_line_lengths.sh`) to the hook so the hard link seems better as it doesn't need admin access, fair enough.